app/controllers/api/versions_controller.rb

```diff
@@ -1,12 +1,41 @@
 class API::VersionsController < API::RestfulController
+  def show
```

Comment:
Cool. This is the most complex controller method in the app at the moment; I'd love to see it
a) slim down some, and
b) move into a service (`VersionService#compose` maybe? Which takes a model and index, and returns a version to return

```
def show
  # (service defaults to the VersionService constant)
  self.resource = service.compose(versions: model.versions.first(index+1))
  respond_with_resource
end
```
then, in the version service:
```
class VersionService
  def self.compose(versions:)
    index = versions.count
    # the existing version reduce / merge code
    # returns a version with composed object_changes
  end
end
```